schedule/yast/autoyast/autoyast_keyboard_layout.yaml

```diff
@@ -5,8 +5,10 @@ description: >
   different than us.
 vars:
   AUTOYAST: autoyast_sle15/autoyast_keyboard_layout.xml
+  AUTOYAST_PREPARE_PROFILE: '1'
   DESKTOP: textmode
   INSTALL_KEYBOARD_LAYOUT: cz
+  INSTALLONLY: '1'
```

Comment:
HDDSIZEGB=30 is needed here as the default is 20 and i guess there is a reason for it. in the same logic HDDMODEL=scsi-hd is not presented in the machine settings, so it has to be here.

Reply:
I have put these short of variable in jobgroups, as they are supposed to be needed before the worker starts running the test.

Reply:
no problem